docs/operating-scylla/security/_common/upgrade-warning-runtime-authentication.rst

```diff
@@ -0,0 +1,20 @@
+
+After Upgrading from 5.4
+----------------------------
```

Comment:
Nit: this probably should be a higher level section (underlined with `^` and not `-`), now it looks like a subsection of `Disable Authentication Without Downtime` which I'm not sure was intended.